backend/src/api/bitcoin/bitcoin-api.interface.ts

```diff
@@ -90,9 +90,10 @@ export namespace IBitcoinApi {
     scriptPubKey: {                  //  (json object)
       asm: string;                   //  (string) the asm
       hex: string;                   //  (string) the hex
-      reqSigs: number;               //  (numeric) The required sigs
+      reqSigs?: number;              //  (numeric) The required sigs
       type: string;                  //  (string) The type, eg 'pubkeyhash'
-      addresses: string[]            //  (string) bitcoin address
+      addresses?: string[]           //  (string) bitcoin addresses
+      address?: string              //  (string) bitcoin address
```

Comment:
You forgot the semicolons.
```suggestion
      addresses?: string[];           //  (string) bitcoin addresses
      address?: string;              //  (string) bitcoin address
```